### DISCUSSION TITLE
The schema of the introspected database was inconsistent: Table User not found.

### DISCUSSION THREAD

**Discussion by lkin** [prisma/prisma#4567] [Q&A] [answered]
Hi! I want to explore prisma 2 using the prisma-examples/typescript/rest-nextjs-api-routes-auth/ template cloned locally. After resolving an initial dependency conflict between React versions 16 and 17 I've followed the instructions [thus far](https://github.com/prisma/prisma-examples/tree/latest/typescript/rest-nextjs-api-routes-auth#2-introspect-your-database), manually editing the database using the sqlite3 sli. However, the `npx prisma introspect` command fails with the following error
```
Error: P4002

The schema of the introspected database was inconsistent: Table User not found.
```
the default prisma/schema.prisma contains:
```prisma
model User {
  id            Int       @default(autoincrement()) @id
  name          String?
  email         String?   @unique
  emailVerified DateTime? @map(name: "email_verified")
  image         String?
  createdAt     DateTime  @default(now()) @map(name: "created_at")
  updatedAt     DateTime  @updatedAt @map(name: "updated_at")
  posts         Post[]

  @@map(name: "users")
}
```

Browsing dev.db with TablePlus I can see the `users` table just fine. What am I doing wrong here? Grateful for any help

**Comment by ryands17** [comment]
Hey @lkin 👋 

Could you remove the `User` model from your `schema.prisma` file and run `prisma introspect` again? This should fetch your table from the DB.

**Comment by fullStackDataSolutions** [comment]
I'm having the same issue even after I deleted the offending table data in the prisma.schema table, is the data cached somewhere?

**Comment by fullStackDataSolutions** [comment, marked as answer]
It was a reference to a table in another schema that was causing the issue for me.

**Reply by janpio** [reply]
In 2.15.0 and newer you should be getting a more specific error message (see https://github.com/prisma/prisma-engines/pull/1493) than the one in the title of this discussion. Wasn't that the case for you or are you on an older version?

**Reply by fullStackDataSolutions** [reply]
I'm on version 2.14, so I will update.

**Comment by ziimakc** [comment]
In my case this was related to table owner issue.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
